cpp/include/cudf/io/parquet.hpp

```diff
@@ -515,6 +515,15 @@ class chunked_parquet_reader {
  * @file
  */
 
+/**
+ * @brief Struct used to describe column sorting metadata
+ */
+struct sorting_column {
+  int column_idx;       //!< leaf column index within the row group
+  bool is_descending;   //!< true if sort order is descending
+  bool is_nulls_first;  //!< true if nulls come before non-null values
```

Comment:
This makes the grammar side of me cringe, but my programmer side so happy at the same time. I'm so conflicted! 😅

Reply:
LOL. I figured someone would insist on `is_` at some point.